src/tests/components/articles.test.jsx

```diff
@@ -31,3 +34,47 @@ describe('AllArticles ', () => {
     expect(component.find('div')).toHaveLength(4);
   });
 });
+
+describe('CreateArticle', () => {
+  it('renders all articles correctly', () => {
+    const onSubmit = jest.fn();
+    const handleChange = jest.fn();
+    const onEditorStateChange = jest.fn();
+    const tags = [{ id: 1, test: 'tags' }];
+    const draftStatus = 'saving';
+    const handleDelete = jest.fn();
+    const handleAddition = jest.fn();
+    const handleDrag = jest.fn();
+    const onPublish = jest.fn();
+    const show = false;
+
+
+    const component = shallow(
+      <CreateArticle
+        onSubmit={onSubmit}
+        handleChange={handleChange}
+        onEditorStateChange={onEditorStateChange}
+        tags={tags}
+        uploadImageCallBack={uploadImageCallBack}
+        draftStatus={draftStatus}
+        handleDelete={handleDelete}
+        handleAddition={handleAddition}
+        handleDrag={handleDrag}
+        onPublish={onPublish}
+        show={show}
```

Comment:
@RachelleMaina, create a `const props = {    const onSubmit = jest.fn();
    const handleChange = jest.fn();
    const onEditorStateChange = jest.fn();
    const tags = [{ id: 1, test: 'tags' }];
    const draftStatus = 'saving';
    const handleDelete = jest.fn();
    const handleAddition = jest.fn();
    const handleDrag = jest.fn();
    const onPublish = jest.fn();
    const show = false;
}` and spread it in `<CreateArticle />` using `<CreateArticle {...props} />`